### ISSUE TITLE
[SUGGESTION] Rotation of Starter Block

### ISSUE THREAD

**Issue by cloudrambler** [tsunamayo/Starship-EVO#3220]
The starter block is great for anchoring partially built ships, but it would be even better if it could be rotated with the ship attached. This way we could orient the area we are working on to the natural light source and not have to deal with the challenge of how different the surfaces look between full light and shadow. Also it is generally more intuitive (as a human) to look down on the thing you are working on, so flipping a hull over to work on the underside would feel better than hovering underneath it.

Pressing [f] on the starter block brings up the blueprint menu by default, even when a blueprint is already loaded. Perhaps this could be changed so that once a blueprint is loaded, the [f] interface instead brings up some other functions; rotate, drag, turn anti-gravity on/off. This would be really helpful for manipulating things that are not complete ships.

**Comment by tsunamayo** [comment]
Yes I think you are right. I was also toying with the idea of having a proper anti-grav tool to manipulate stuff like in garry's mod as suggested by other people. Thx
